cmd/krun/krun.go

```diff
@@ -28,6 +29,13 @@ import (
 var initDebug func(run *mesh.KRun)
 
 func main() {
+	// If ENABLE_TRAFFIC_DIRECTOR env variable is set then only set up necessary
+	// to conenct to Traffic Director control plane will be taken.
+	if os.Getenv("ENABLE_TRAFFIC_DIRECTOR") != "" {
```

Comment:
Can we use MESH env variable, and startsWith("td:") ?

Reply:
Introduced with @0cc507005720c41ef8e4044813332df1ce78337b PTAL.